### DISCUSSION TITLE
Taxa cannot be aggregated at any rank

### DISCUSSION THREAD

**Discussion by rauldenia1** [david-barnett/microViz#142] [Q&A] [answered]
Hello, i cant use my phyloseq object because it says taxa cannot be agreggated at rank: (im trying all ranks and only works with ASV, but im not interested in that)
It says that i have to use tax_fix but it displays all the rank names, and i fixed the table but it doesnt work

Thank you for your help :)

**Comment by david-barnett** [comment]
hi! can you perhaps paste your code and the error message you get for e.g. aggregating at genus level?

**Comment by rauldenia1** [comment, marked as answer]
Hello! I have just solved the problem. The problem was that my taxa table had a column with the sequences of the ASVs so i deleted that column and now it works perfectly. Thanks anyway :)
